projects/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.1 on 2022-08-17 14:06
+# Generated by Django 4.0.5 on 2022-09-07 09:29
```

Comment:
This is actually unexpected and we should talk about it. It should not be possible to auto-generate the generations with two different Django versions. We have to improve the developer setup. 

Apart from that, there is something wrong with the merge from `main` to this branch. The changes of this generations are not yet present on `main`. Was this merged from a stale branch locally ?